graalpy-notebook-example/src/main/java/com/oracle/example/javapython/Main.java

```diff
@@ -1,7 +1,44 @@
 /*
- * Copyright (c) 2021 Oracle and/or its affiliates.
- * Copyright (c) 2021 Software Architecture Group, Hasso Plattner Institute
```

Comment:
@LesiaChaban, add this line above Oracle's copyright. Keep the year as it is. 
`* Copyright (c) 2021 Software Architecture Group, Hasso Plattner Institute`